README.md

```diff
@@ -0,0 +1,84 @@
+# PreCICE Config Graph
+
+A Library that builds a graph from a PreCICE configuration file for validation and visualization purposes.
```

Comment:
Maybe add here that it only checks logical (non functional) errors and that before running any precice project the current checker should also be executed (i.e. precice-tools check).